### ISSUE TITLE
Could we have random maint loot on Box?

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#4523]
Bit of a discussion, bit of a feature request.  

Something I think Deff does really well is the random loot in maint.  Does anyone else feel the same way?  It mixes up the "Rush to the same spot to get the same coin" aspect of the game and you can have more interesting items if they aren't a fixed, static spawn that people will always go for.  

Maybe it's a t-ray scanner or maybe it's insulated gloves, or maybe they're just black gloves that were colored yellow with a crayon that are merely pretending to be insulated gloves.

I really like that aspect of Deff.  Does anyone else think it should be (on a limited basis) added to Box?

**Comment by TermedClepe** [comment]
Why nerf assistants?

**Comment by CptWad** [comment]
Why NOT nerf assistants. Less assistants mean more security or engineers.

**Comment by Probe1** [comment]
The idea isn't to nerf assistants at all.  The idea is to give them even better stuff but not at fixed positions so they can memorize it and rush it at round start.

**Comment by 9600bauds** [comment]
Removing map meta-knowledge is a nerf?
Hold on, let me rephrase that question.
You hear "random maint loot" and you immediately interpret it as an assistant nerf, specifically?

**Comment by TermedClepe** [comment]
>Removing map meta-knowledge is a nerf?
Yes, think removing guaranteed rare ore deposits from clown asteroid / western mining field.
>you immediately interpret it as an assistant nerf, specifically?
Every job has a department for their items, except assistants. Nerfing their only 'access' area is indeed specifically nerfing assistants.

**Comment by Exxion** [comment]
Box has SOME random maint loot, but not very much and it's hidden in small sealed rooms
It has a chance to spawn antag-only items, too, for some reason

Honestly, anything that would make the other maps' maints more like Defficiency maint would be good
Except spiders

**Comment by SangriaMonkey** [comment]
Dude we should add bears to every secret room so anyone trying to metamappowerbuttgame gets shredded to bits. And anyone who enters the tunnel afterward. Balance.

**Comment by mrpain666** [comment]
Add some risk vs reward. Put some hostile mobs in there if people are able to get antag only items. Like space bears. Or mice that bite and deal toxin damage and make you sick or something. Some randomization wouldnt hurt either.

**Comment by Probe1** [comment]
Not antag items.  Not crazy fucking plasma cannons.  Just changing instead of 'a t-ray scanner it might be insulated gloves'.  Or instead of a firefighting suit it might be a space suit.  Or something worse.  Instead of Meson goggles they might be sunglasses, or welding goggles.  It might be great and give you a head start on your gray tide escapades or it might be situational that maybe an engineer would want but an assistant wouldn't, or it might be complete crap that no one wants.  

The point is you don't know until you go there and see.

>diseased mice
although I think it'd be funny and kinda cool for mice to have viruses and it might actually make 'bioscans report mice breeding in the library' a meaningful random event, I'm just talking about random loot like on Deff.

**Comment by NigglyWiggly** [comment]
>Hostile mobs in the maintenance tunnels of a high-end space station

**Comment by Probe1** [comment]
ye I didn't reply to that because I didn't think I had to.

@mrpain666 we tried that on Deff and the playerbase crucified Duny for it.  People did not like it at all.  I think it's a nice idea for off station and it technically fit the mood of Deff but it just did not go over well and that's the bottom line.

I'm narrowly wondering if having random items in some places of maint is better than static items.  Something Velard discussed and interested in working on but ultimately did not was randomizing z levels so you could afford to put valuable items out there.  Currently, the consensus is putting valuable items on the derelict, telecomms satellite and other locations fucking sucks because people will rush it every round just like people do now for the suspicious toolbox to get cheap illegal research.  If items are randomized then they can both be valuable and not immediately result in memorization of spawn locations and a variant of powergaming.

-Does it make exploring maint more enjoyable? 

-Does the risk/reward of checking many places for better items give more enjoyment than going to a precise location as fast as you can to get an average item?

These are the questions I feel must be answered.  So far only @TermedClepe has answered the questions and I'd like to read a wider range of opinions of any length.

**Comment by ComicIronic** [comment]
>nerf assistants

Ho boy. I too am worried by the balance implications of not being able to rush the same silver coin in maintenance every round.

I like it as an idea. It would degame-ify maintenance exploration, and make it feel like "oh, what if I could find x in here". If you're really worried about the `balance` involved, you could make it so that certain items in certain amounts are guaranteed to show up.

**Comment by dylanstrategie** [comment]
We already have Maintenance spawners entities, but they need to be clearly positioned. Besides, that wouldn't stop Assistant loot raiding, just make it more unpredictable

Also nerfing Assistants, if that's even a thing, is a big no-no. We're already making sure you don't get a massive greytide with Security to go with it

**Comment by TermedClepe** [comment]
@ComicIronic

> I too am worried by the balance implications of not being able to rush the same silver coin in maintenance every round.

You don't seem to be worried about science rushing the same research order or miners mining the same location. In fact you considered it "controversial" to so much as make lockboxes less of a joke.

I think you're very selective and biased when it comes to balancing.

**Comment by ComicIronic** [comment]
You misunderstand. I do not think this is a balance issue at all.
There is no balance design as to what spawns where in maint. The only consideration made is that the items should not be too useful - real yellow gloves, spacesuits, etc - but other than that it fails to impact the game.

Mining is also a poor comparison. Ores don't spawn in the same place every round. Most of the asteroid is determined by RNG - only the location of the bases, the high-yield field, and the shape of the asteroid itself are truly fixed.

**Comment by TermedClepe** [comment]
>I do not think this is a balance issue at all.

That's exactly my point. You don't consider a single coin available at roundstart being able to impact a round.

>Ores don't spawn in the same place every round

Every round there's diamonds without gibtonite, gold, uranium, silver and plasma all in the same small area.

**Comment by ComicIronic** [comment]
And every round similar equipment could spawn in similar places in maintenance.
It doesn't have to be the same each round.

>You don't consider a single coin available at roundstart being able to impact a round.

That's not the same as being a design issue. Anything can impact a round with the right chances.

**Comment by TermedClepe** [comment]
Should such a system be implemented, there still needs to be a way to control distribution of items aside from "x % chance at y spot". So for example there aren't 5 tray scanners but no belts to be found.
